boltdesignsystem/bolt
# Card | Docs | Add demo of controlling horizontal card image size

## Jira

N/A

## Summary

Updates the horizontal card's doc page to show usage of card media width.

## How to test

Run the branch locally and check the horizontal card documentation page. Make sure there's no typos.
